Src/Helpers/domInteractions.ts

```diff
@@ -0,0 +1,48 @@
+import { randomIntFromInterval } from "./mathHelper.js"
```

Comment:
you should import from "./mathHelper" as long as that is a module.

Reply:
I used Webpack to implement this.